common/views/components/IIIFViewerPrototype/ViewerTopBarPrototype.tsx

```diff
@@ -228,26 +226,38 @@ const ViewerTopBar: FunctionComponent<Props> = ({
       <Main>
         <LeftZone>
           {!showZoomed && canvases && canvases.length > 1 && (
-            <ShameButton
-              className={`viewer-desktop`}
-              isDark
-              ref={viewToggleRef}
-              onClick={() => {
-                setGridVisible(!gridVisible);
-                trackEvent({
-                  category: 'Control',
-                  action: `clicked work viewer ${
-                    gridVisible ? '"Detail view"' : '"View all"'
-                  } button`,
-                  label: `${work.id}`,
-                });
-              }}
-            >
-              <Icon name={gridVisible ? 'detailView' : 'gridView'} />
-              <span className={`btn__text`}>
-                {gridVisible ? 'Detail view' : 'View all'}
-              </span>
-            </ShameButton>
+            <ToolbarSegmentedControl
+              hideLabels={true}
+              items={[
+                {
+                  id: '1',
```

Comment:
Worth using something more descriptive e.g. `page` & `grid` for ease of reading in the conditions?